packages/ckeditor5-list/docs/features/lists-editing.md

```diff
@@ -0,0 +1,69 @@
+---
+menu-title: Lists editing bahavior
+meta-title: Lists editing bahavior | CKEditor 5 Documentation
+category: features-lists
+order: 30
+modified_at: 2024-01-02
+---
+
+# Lists editing behavior
+
+## Block lists
+
+The current list feature lets any part of the content be part of a list. Content blocks and elements – such as blockquotes, tables, paragraphs, headings, and others – can be put inside a list item, ensuring the continuity of numbering and retaining indentation.
+
```

Comment:
1.  I would love to accompany the text with a GIF, like in the `Typing around links`. This could show, that pressing backspace inserts inside the list. I noticed people don't understand it usually.
2.  Blockquote needs to be removed, this is actually a known feature request/bug. It could be replaced with images.

Reply:
2\. In commit.